classes/app/exhibition/ExhibitionBoundary.php

```diff
@@ -31,4 +31,22 @@ public function deleteExhibition(Delete\Request $request)
         $deleteExhibition = new Delete\DeleteExhibition($this->authenticator, $this->exhibitionRepository);
         $deleteExhibition->delete($request);
     }
+
+    public function listAllExhibitions(ListAll\Request $request)
+    {
+        $listAllExhibition = new ListAll\ListAllExhibitions($this->exhibitionRepository);
+        $response = new ListAll\Response();
+        $response->exhibitions = $listAllExhibition->listAll($request);
+
+        return $response;
+    }
+
+    public function getExhibition(Get\Request $request)
+    {
+        $getExhibition = new Get\GetExhibition($this->exhibitionRepository);
+        $response = new Get\Response();
```

Comment:
The same applies here: the `Response` object should be created within the use-case itself.